python/tvm/contrib/hexagon/session.py

```diff
@@ -44,7 +44,9 @@ class Session:
         Remote configs for RPC tracker.
 
     session_name : str
-        Hexagon RPC session name.
+        Hexagon RPC session name. Options are [hexagon-rpc, cpu-rpc].
```

Comment:
I'm somewhat concerned with the change in semantics here.  Going from a human-readable name that can be arbitrarily specified to a list of specific options feels like there's got to be a better way.